### PR TITLE
Task-45811 Update Profile and space avatars in chat application when there is updating in the platform

### PR DESCRIPTION
**Pull request by SaraBoutej**
Task-45811 Update Profile and space avatars in chat application when there is updating in the platform

When we update a profile or a space avatars in the platform, they are not updated in the chat room, so we update the user and space Avatars URl in all chat application

### CODE REVIEW

File: application/src/main/webapp/vue-app/components/ExoChatContact.vue [hunk 1]
```diff
@@ -173,9 +175,11 @@ export default {
     },
     contactAvatar() {
       if (this.type === 'u') {
-        return getUserAvatar(this.userName);
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/ExoChatContact.vue:176]: shouldn't we remove old functions that retrieve avatars ?

Comment (SaraBoutej) [reply application/src/main/webapp/vue-app/components/ExoChatContact.vue:176]: The Old function return return `${chatConstants.SOCIAL_USER_API}${user}/avatar`;
we need to retrieve the new avatars with the lastModified attribute /portal/rest/v1/social/users/root/avatar?lastModified=1630604606385&r=uz1dBGKQFq2OiTlaw89eErNF5P%2FsLsMtr0tZSYxhczgRlpmi1Wz6auFUgjOsMNs0
to update it dynamically in the chat application

cc @ahamdi

Comment (ahamdi) [reply application/src/main/webapp/vue-app/components/ExoChatContact.vue:176]: Are this function still useful ? if not, let's remove it 
 https://github.com/exoplatform/chat-application/blob/3887f8ee43a0bf7868d16b37116413b09d89322c/application/src/main/webapp/vue-app/chatServices.js#L384

**Comment by GouadriaHanen** [comment]
PR updated cc @ahamdi

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**